lms/static/scripts/frontend_apps/components/dashboard/ContentWithBadge.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ComponentChildren } from 'preact';
+
+export type ContentWithBadgeProps = {
+  count: number;
+  /** Indicates we know the count could increase. Defaults to false */
+  hasMoreElements?: boolean;
```

Comment:
```suggestion
  hasMoreItems?: boolean;
```

We are mixing the terms "items" and "elements" to refer to the number of entries in the list. I suggest using the term "items" because are talking about logical items in a list, rather than DOM elements. There happens to be one DOM element per rendered item, but that's an implementation detail.

Reply:
Yeah, yeah. `hasMoreItems` makes way more sense.